react/src/App.js

```diff
@@ -1,8 +1,8 @@
 import { Route, Routes } from 'react-router-dom';
-import HomePage from './pages/Home';
-import Menu2Page from './pages/Menu2';
-import Menu3Page from './pages/Menu3';
-import PogodaPage from './pages/Pogoda';
+import HomePage from '@pages/Home';
+import Menu2Page from '@pages/Menu2';
+import Menu3Page from '@pages/Menu3';
+import PogodaPage from '@pages/Pogoda';
```

Comment:
Bardziej miałem na myśli `@/pages/Pogoda`, czyli `@` = `src`. Dasz radę przerobić?

Reply:
Ah, no jasne! Nawet tak miałem wcześniej, ale pomyślałem że jak skracać to na maksa 😅